scripts/explorer-jes-start.sh

```diff
@@ -31,6 +31,10 @@ SERVER_DIR="${ROOT_DIR}/components/explorer-jes/bin/server/"
 
 JOB_NAME="${ZOWE_PREFIX}UJ"
 
+if [ -z "${ZOWE_EXPLORER_FRAME_ANCESTORS}" ]; then
+  ZOWE_EXPLORER_FRAME_ANCESTORS="${ZOWE_EXPLORER_HOST}:*,${ZOWE_IP_ADDRESS}:*"
```

Comment:
ZOWE_EXPLORER_HOST and ZOWE_IP_ADDRESS should be added to the top of the file under the required shell variables